setup.py

```diff
@@ -10,7 +10,8 @@
     packages=find_packages(),
     license='ISC',
     install_requires=[
-        'ruamel.yaml'
+        'ruamel.yaml',
+        'rich~=13.2.0'
```

Comment:
Until there is a /known/ compatible problem, I disagree with specifying the version. It's false specificity and generates future work.

Reply:
As per our discussion on this topic, I have changed this to only keep the major version in place. If later versions cause no deprecation warnings, issues, or failing tests the package version can be incremented manually.